authentication/certificates/service-serving-certificate.adoc

```diff
@@ -0,0 +1,27 @@
+[id="add-service-serving"]
+= Service serving certificate secrets
```

Comment:
This is titled like a pure conceptual assembly but includes procedures. Please change the title.

I think this assembly should be broken into two, one about setting and one about manually rotating. Please check and make sure that you have one user story per assembly.

Reply:
I've adjusted the title to be `Securing service traffic using service serving certificate secrets`. The goal here is to secure inter-cluster traffic between services.

While an argument could be made about this being two user stories, I think this is really one story - securing service traffic. If you secure service traffic, then you must be aware of the service CA certificate's expiration and how to rotate it if needed. As mentioned in another comment, I strongly suspect few customers will ever need those steps, but I don't believe it belongs in its own section.

I'm leaving this as one assembly for now.